examples/account/create_vc.rs

```diff
@@ -0,0 +1,101 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! A basic example that generates and publishes subject and issuer DID
+//! Documents, then creates a Verifiable Credential (vc) specifying claims about the
+//! subject, and retrieves information through the CredentialValidator API.
+//!
+//! Note: This example uses in-memory storage for simplicity.
+//! See `create_did` example to configure Stronghold storage.
+//!
+//! cargo run --example create_vc
+
+use identity::account::Account;
+use identity::account::IdentitySetup;
+use identity::account::MethodContent;
+use identity::account::Result;
+
+use identity::core::json;
+use identity::core::FromJson;
+use identity::core::ToJson;
+use identity::core::Url;
+use identity::credential::Credential;
+use identity::credential::CredentialBuilder;
+use identity::credential::Subject;
+use identity::crypto::ProofOptions;
+use identity::did::DID;
+use identity::iota::CredentialValidationOptions;
+use identity::iota::CredentialValidator;
+use identity::iota::FailFast;
+
+mod create_did;
+
+pub async fn create_vc() -> Result<String> {
+  // Create an identity for the issuer.
+  let mut issuer: Account = Account::builder().create_identity(IdentitySetup::default()).await?;
```

Comment:
We should create one `AccountBuilder` and use it to create both `issuer` and `alice` here, unless that is intentional.